pkg/result/processors/autogenerated_exclude.go

```diff
@@ -43,8 +43,8 @@ func (p *AutogeneratedExclude) Process(issues []result.Issue) ([]result.Issue, e
 
 func isSpecialAutogeneratedFile(filePath string) bool {
 	fileName := filepath.Base(filePath)
-	// fake files to which //line points to for goyacc generated files
-	return fileName == "yacctab" || fileName == "yaccpar" || fileName == "NONE"
+	// fake files or generation definitions to which //line points to for generated files
+	return filepath.Ext(fileName) != ".go"
```

Comment:
maybe `filepath.Ext(filePath) != ".go"` is good enough.

Reply:
Actually, if we are comparing the full path we can avoid `filepath` and skip calling into OS specific code and just do `strings.HaveSuffix(filePath, ".go")`. Maybe we can save a microsecond or three? 😄